README.md

```diff
@@ -26,59 +27,59 @@ rustup component add rustfmt                # Install the code auto-formatter.
 rustup component add clippy                 # Install the linter.
 ```
 
-### Building and testing the project
-
-Please use the `script/build.sh`, `script/watch.sh`, and `script/lint.sh`
-scripts to build, watch, and lint the project respectively. We need to use a
-simple custom wrappers here because of the several Rust toolchain issues:
-
-- [No direct support for Cargo Workspaces in
-wasm-pack.](https://github.com/rustwasm/wasm-pack/issues/642). Fixed in
-`build.sh`. 
-- There is no watch utility in wasm-pack, which makes using it harder than it
-should be. Fixed in `watch.sh`.
-- [The commands cargo-check and cargo-clippy do not clean local cache if used
-several times.](https://github.com/rust-lang/cargo/issues/6986). Fixed in
-`lint.sh`.
-
-In order to build an example demo scene, please use the following commands:
+### Building the sources
+Please use the `script/build.sh` script to build the project or the
+`script/watch.sh` script to run a file watch utility which will build the
+project when on every source change. The scripts are thin wrappers over
+[wasm-pack](https://github.com/rustwasm/wasm-pack) and accept the same [command
+line arguments](https://rustwasm.github.io/wasm-pack/book/commands/build.html).
+In particular, you can provide them with `--release`, `--dev`, or `--profile`
+flags to switch the compilation profile. If not option is provided, the scripts
+default to the `--release` profile.
+
+### Running examples
+Please note that in order to run the examples you have to first build the
+project. For best experience, it is recommended to use the `scripts/watch.sh`
+in a second shell. In order to build the demo scenes, follow the steps below:
 
 ```bash
-./script/watch.sh # Build and watch for changes.
-
-# Wait till the project finishes building.
-# Run the following lines from other cmd:
-
-cd examples/01-scene
+cd examples
 npm install
-npm run start
+npm run start 
 ```
 
-You can now open the following address in your browser: http://localhost:8080.
+You can now navigate to http://localhost:8080 and play with the demo scenes!
 
-There are also _web test_ which are run in browser and produce some output. To
```

Comment:
Why this section was deleted? I think this information should be available somewhere

Reply:
3 reasons:
1. All scripts should be short commands, if you want to put something to readme, create a script for it in scripts.
1. It sounds not very professional to me. "There are also _web test_ which are run in browser and produce some output". Its like - "yeah there are some tests and some output, yo". 
2. If I understand correctly, the `./scripts/test` contain those, right? 

Anyway, I'd be happy to add more info about it, ut I need help with it.

Reply:
Ad 3. no they don't. The script run all tests in headless mode. And if you need for short script for that, it should be a part of this PR, or a task should be created for this.

Reply:
As we agreed on phone call, ill add the script in next pr.